### ISSUE TITLE
defining an external prop failing

### ISSUE THREAD

**Issue by mhuebert** [lezer-parser/lezer#33]
I'm trying to define an external prop, following the guide (https://lezer.codemirror.net/docs/guide/#node-props):

```
// declare external prop
@external prop myProp from "./props"
// ...later...
Set[myProp=true] { "#" Map }
```

When running this, I see the error `Unknown prop name 'myProp'`. 

To set up the parser I'm using the `buildParser` function and passing in an `externalProp` function, but this function does not seem to be called. My code is roughly:

```js
buildParser(clojureGrammer, 
            {externalProp: 
              function(propName) 
                { // should return an instance of NodeProp
                  // (but this code is never evaluated)
                  if propName == 'myProp' { return NodeProp.flag() }
                 }})
```

Any ideas on what I might be missing?

**Comment by marijnh** [comment]
That was a bug in the library. Fixed in e8d0c2eba30d.  lezer-generator 0.10.4 should work better.

**Comment by mhuebert** [comment]
Thanks for the fix!
